File: app/src/main/java/com/bopr/android/smailer/ui/PhoneWhitelistActivity.java
package com.bopr.android.smailer.ui;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

/**
 * Phone number whitelist activity.
 *
 * @author Boris Pronin (<a href="mailto:boprsoft.dev@gmail.com">boprsoft.dev@gmail.com</a>)
 */
public class PhoneWhitelistActivity extends AppActivity {

    @NonNull
    @Override
    protected Fragment createFragment() {
        return new PhoneWhitelistFragment();
    }

}

File: app/src/main/kotlin/com/bopr/android/smailer/ui/PhoneWhitelistActivity.kt
package com.bopr.android.smailer.ui

import androidx.fragment.app.Fragment

/**
 * Phone number whitelist activity.
 *
 * @author Boris Pronin ([boprsoft.dev@gmail.com](mailto:boprsoft.dev@gmail.com))
 */
class PhoneWhitelistActivity : AppActivity() {

    override fun createFragment(): Fragment {
        return PhoneWhitelistFragment()
    }
}